objcxx_eh.cc

```diff
@@ -515,14 +519,71 @@ static void eh_cleanup(void *exception)
 	objc_release(*(id*)exception);
 }
 
+namespace __cxxabiv1
+{
+    struct __cxa_exception
+    {
+		// Manage the exception object itself.
+		std::type_info *exceptionType;
+		void (*exceptionDestructor)(void *);
+		terminate_handler unexpectedHandler;
+		terminate_handler terminateHandler;
+		__cxa_exception *nextException;
+		int handlerCount;
+		int handlerSwitchValue;
+		const unsigned char *actionRecord;
+		const unsigned char *languageSpecificData;
+		void *catchTemp;
+		void *adjustedPtr;
+		_Unwind_Exception unwindHeader;
+    };
+
+	struct __cxa_refcounted_exception
```

Comment:
Not all runtimes have this, and it causes different offsets because of alignment padding.

Reply:
Correct, but MSYS2 really only ships with two runtimes -- libstdc++ and libc++.